tests/testthat/test-setup_docs.R

```diff
@@ -61,6 +61,5 @@ test_that("quarto: README.qmd", {
   create_local_package()
   expect_false(file.exists("README.qmd"))
   setup_docs("quarto_website", overwrite = TRUE)
-  expect_true(file.exists("README.qmd"))
+  expect_false(file.exists("README.qmd"))
```

Comment:
If there is nothing, we should return a Quarto document, because we are building a Quarto website and want users to be able to use all features on the home page.

I agree, of course, that this should be customizable and that users who want just README.md should be free to change this.

Reply:
Related to the above, I don't think Quarto changes anything by file extension. (Again, why would a user who prefers ipynb want to use Quarto CLI if there are features that are only available in qmd?)